proto/umee/leverage/v1beta1/tx.proto

```diff
@@ -6,16 +6,18 @@ option go_package = "github.com/umee-network/umee/x/leverage/types";
 import "cosmos/base/v1beta1/coin.proto";
 import "gogoproto/gogo.proto";
 
-// MsgLendAsset represents a lender's request to lend a base asset type to the module
+// MsgLendAsset represents a lender's request to lend a base asset type to the
```

Comment:
@toteki if you use VScode, I recommend installing clang-format so that you can format proto files on save 👍

Reply:
Got it